test/e2e/v1beta1/scripts/gh-actions/run-e2e-experiment.py

```diff
@@ -249,6 +255,11 @@ def run_e2e_experiment(
     if experiment.metadata.name == "random":
         MAX_TRIAL_COUNT += 1
         PARALLEL_TRIAL_COUNT += 1
+        if args.trial_pod_annotations:
+            trial_spec = experiment.spec.trial_template.trial_spec
+            trial_spec_metadata = trial_spec['spec']['template'].get('metadata', {})
+            trial_spec_metadata['annotations'] = eval(args.trial_pod_annotations)
```

Comment:
What format do you plan to pass? Also, Consider a case where annotations already present

Reply:
I'm planning to pass string representation of a python dictionary (annotations's key-valye pairs) e.g. I'll pass `'{"sidecar.istio.io/inject": "false"}'` to disable sidecar injection.

Will update the code to consider already present annotations.